subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r25/TestProgressCrossVersionSpec.groovy

```diff
@@ -509,6 +509,7 @@ class TestProgressCrossVersionSpec extends ToolingApiSpecification {
     def goodCode() {
         buildFile << """
             apply plugin: 'java'
+            sourceCompatibility = 1.6
```

Comment:
Looks like previous Gradle used an old version ASM which couldn't recognize 1.8 bytecode. This does occur on my local machine `crossVersionTests`, I don't understand why it's never fixed.

Reply:
Our cross-version tests currently run on Java 7, that's probably why.

Reply:
Good to know. That should be the reason.